core/design-system/src/main/java/com/moneymong/moneymong/design_system/component/datepicker/MDSBottomSheetDatePicker.kt

```diff
@@ -294,47 +317,49 @@ internal fun MDSDatePickerDateView(
             Spacer(modifier = Modifier.height(2.dp))
             Text(
                 text = "${year}년 ${month}월",
-                style = Heading2,
+                style = Heading3,
                 color = if (!isValidValue && selected) Red03 else Blue04
             )
         }
     }
 }
 
-@OptIn(ExperimentalFoundationApi::class)
 @Composable
 internal fun <T> WheelPicker(
     modifier: Modifier = Modifier,
     state: DatePickerState = rememberMDSDatePickerState(),
-    size: DpSize = DpSize(width = 64.dp, height = ((Heading2.lineHeight.spToDp * 3) + (28.dp * 2))),
+    size: DpSize = DpSize(width = 64.dp, height = 140.dp),
     items: List<T>,
     startIndex: Int = 0,
     visibleItemsCount: Int = 3,
     confirmDateChange: (value: T) -> Unit = {},
     content: @Composable LazyItemScope.(item: T, color: Color) -> Unit
 ) {
-    val lazyListState =
-        rememberLazyListState(startIndex - 1) // empty string을 위해 리스트 처음과 끝 값을 제외함
+    val paddedItems = listOf(null) + items + listOf(null)
+    val safeStartIndex = startIndex.coerceIn(0, (items.size - 1))
```

Comment:
아래처럼 하면 더 깔끔할 듯 합니다!

```suggestion
    val safeStartIndex = startIndex.coerceIn(0, (items.lastIndex))
```